src/main.c

```diff
@@ -1,10 +1,13 @@
 #include "utilities/sdl_functions.h"
 #include "ui/menu_window.h"
+#include "models/audio.h"
 
 int main(void)
 {
     initSDL();
+    initSDL_Mixer();
 
+    initAudio();
```

Comment:
I think that we can also init pacman, ghosts, fruits & timers here.
Playing a game won't change what is initialized by this function so we can only initialize this once instead of at the start of each game :)

This way, we can initialize theses

Reply:
(this is not specific to this we, we can do this in another branch)

Reply:
What a good idea my dear fellow ! I will create an issue for this.

Reply:
https://github.com/Livl-Corporation/livl-pacman/issues/79